js/net_test.ts

```diff
@@ -103,24 +119,25 @@ testPerm({ net: true }, async function netListenAsyncIterator(): Promise<void> {
   conn.close();
 });
 
-/* TODO Fix broken test.
-testPerm({ net: true }, async function netCloseReadSuccess() {
+testPerm({ net: true }, async function netCloseReadSuccess(): Promise<void> {
```

Comment:
The networking tests are currently (erroneously) disabled. I want to land https://github.com/denoland/deno/pull/2268 first. But I this PR LGTM - thanks.